### DISCUSSION TITLE
Filter search results based on files number (mfif)

### DISCUSSION THREAD

**Discussion by Cmely** [nicotine-plus/nicotine-plus#2488] [Q&A] [answered]
With Soulseek it's possible to filter search result with the "minfilesinfolder" (or "mfif") attribute. This is handy when you're looking for complete album and don't want any single file result. 

Is it possible to achieve the same thing with Nicotine+ ?

:)

**Comment by slook** [comment, marked as answer]
Possibly related: https://github.com/nicotine-plus/nicotine-plus/issues/1023

> Is it possible to achieve the same thing with Nicotine+ ?

Perhaps, but it is not straight forward. It would be necessary to do a FolderContentsRequest for each returned result, in order to count the total number of contained files, that will be very complex.

**Reply by Cmely** [reply]
> Possibly related: #1023

Yes, this is exactly related ;) !
Thank you for the answer :).
